core/src/test/java/org/elasticsearch/index/search/geo/GeoPointParsingTests.java

```diff
@@ -198,4 +200,10 @@ public static void assertCloseTo(final GeoPoint point, final double lat, final d
         assertEquals(point.lat(), lat, TOLERANCE);
         assertEquals(point.lon(), lon, TOLERANCE);
     }
+
+    @Override
+    protected XContent xContentForParser(BytesReference data) {
```

Comment:
I am just wondering did you consider passing xContent type as a parameter here instead of overloading it on the suite level? I feel like it might provide more flexibility.